src/CacheInterface.php

```diff
@@ -2,12 +2,22 @@
 
 namespace React\Cache;
 
+use React\Promise\PromiseInterface;
+
 interface CacheInterface
 {
-    // @return React\Promise\PromiseInterface
+    /**
+     * @return PromiseInterface
+     */
     public function get($key);
 
+    /**
+     * @return PromiseInterface
+     */
     public function set($key, $value);
 
+    /**
+     * @return PromiseInterface
```

Comment:
I think @maciejmrozinski proposed that as a workaround because there's no convention/standard for that kind of placeholder object yet. It is not meant for code completion but as a hint for developers so that they know the resolution value of the promise.